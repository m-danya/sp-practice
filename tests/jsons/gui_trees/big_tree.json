{
  "activity_name": "random_activity",
  "activity": {
    "root": {
      "class": "class1",
      "clickable": false,
      "bounds": [
        0,
        0,
        400,
        500
      ],
      "children": [
        {
          "class": "class1",
          "clickable": false,
          "children": []
        },
        {
          "class": "second_class",
          "clickable": true,
          "bounds": [
            0,
            0,
            100,
            50
          ],
          "visible-to-user": true,
          "children": [
            {
              "class": "class3",
              "clickable": false,
              "children": []
            }
          ]
        },
        {
          "class": "second_class",
          "clickable": true,
          "visible-to-user": true,
          "bounds": [
            0,
            0,
            100,
            50
          ],
          "children": []
        },
        {
          "class": "second_class",
          "clickable": true,
          "bounds": [
            0,
            0,
            100,
            50
          ],
          "visible-to-user": true,
          "children": []
        },
        {
          "class": "second_class",
          "clickable": true,
          "bounds": [
            0,
            0,
            100,
            50
          ],
          "visible-to-user": true,
          "children": []
        }
      ]
    }
  }
}