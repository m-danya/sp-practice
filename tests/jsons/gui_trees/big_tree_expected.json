{
    "count": 7,
    "clickable_elements": 4,
    "activity_name": "random_activity",
    "element_names": {
        "class1": 2,
        "second_class": 4,
        "class3": 1
    },
    "clickable_areas": []
}